front/src/components/Login/Login.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+import React, { Component } from 'react';
+import { connect, type Dispatch } from 'react-redux';
+import { withRouter, RouterHistory } from 'react-router-dom';
+import { TextField, Button } from '@material-ui/core';
+import { loginUser } from '../../actions/authAction';
+import './Login.css';
+
+type Props = {
+    errors: Array<Error>,
+    loginUser: Dispatch,
+    history: RouterHistory
+};
+type State = {
+    email: string,
+    password: string,
+    errors: Object
+};
+
+class Login extends Component<Props, State> {
+    state = {
+        email: '',
+        password: '',
+        errors: {}
+    };
+
+    handleChange = (name: string) => (e: SyntheticEvent<HTMLInputElement>) => {
```

Comment:
please don't use arrow functions. use normal functions and use the constructor to bind(this) if needed.

Reply:
why?